src/NLog/LayoutRenderers/ExceptionLayoutRenderer.cs

```diff
@@ -382,6 +419,21 @@ protected virtual void AppendSource(StringBuilder sb, Exception ex)
 #endif
         }
 
+        /// <summary>
+        /// Appends the HResult of an Exception to the specified <see cref="StringBuilder" />.
+        /// </summary>
+        /// <param name="sb">The <see cref="StringBuilder"/> to append the rendered data to.</param>
+        /// <param name="ex">The Exception whose HResult should be appended.</param>
+        protected virtual void AppendHResult(StringBuilder sb, Exception ex)
+        {
+#if NET45
+            if (ex.HResult != 0 && ex.HResult != 1)
```

Comment:
0 are 1 are special? Is that somewhere documented? Maybe reference here? 

I only see 0 here: https://docs.microsoft.com/en-us/windows/win32/seccrypto/common-hresult-values

Reply:
we could use reflection for older .NET :angel:

Reply:
Added const int documentation